format/DATASET_FORMAT.md

```diff
@@ -18,6 +18,13 @@ A dataset is a JSON object that contains the following components:
 
   # (Optional) Map[char -> Values] - The initial estimate for the values for each robot in the dataset (used for batch algorithms)
   "initialization": {"a": {...}, "b": {...}}
+
+  # (Optional) [List-of-FactorId] - The set of factors that are potentially outliers (i.e. Loop Closures)
+  # If not included all measurements are assumed to be known inliers
+  "potential_outlier_factors": [ [0, 2], [4, 0] ...]
+
+  # (Optional) [List-of-FactorId] - The set of ground truth outlier factors
+  "outlier_factors": [[4, 0], ...]
```

Comment:
In the dataset I believe this are stored as Map[char -> Set[FactorId]] correct? Are they saved in file as just a list?